crates/fluvio-cli/src/client/consume/mod.rs

```diff
@@ -781,3 +772,99 @@ mod cmd {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::ConsumeOpt;
+
+    fn get_opt() -> ConsumeOpt {
+        ConsumeOpt {
+            topic: "TOPIC_NAME".to_string(),
```

Comment:
this probably can be shortened into:
```
ConsumeOpt {
..Defaults::default()
}
```

Reply:
compiling 
```
        ConsumeOpt {
            topic: "TOPIC_NAME".to_string(),
            ..Default::default()
        }

```
gives `the trait std::default::Default is not implemented for ConsumeOpt`. Unless I'm using the syntax wrong. Don't think it's good to implement `Default` for `ConsumeOpt` though.

Reply:
There must be API to create options

Reply:
> There must be API to create options

I assume you meant to post this on the clap thread and resolve this thread.